pkg/operator/ceph/file/mds/mds.go

```diff
@@ -218,8 +313,7 @@ func (c *Cluster) scaleDownDeployments(replicas int32, desiredDeployments map[st
 			logger.Infof("Deleting extraneous mds deployment %s", d.GetName())
```

Comment:
This logs that it will delete deployments when it will actually just scale them down. Both `deleteMdsDeployment` and `scaleMdsDeployment` log what they are doing, so I think you can just remove this line.